central/networkbaseline/manager/manager_impl.go

```diff
@@ -919,6 +920,123 @@ func (m *manager) CreateNetworkBaseline(deploymentID string) error {
 	return nil
 }
 
+func (m *manager) GetExternalNetworkPeers(ctx context.Context, deploymentID string, query string, since *time.Time) ([]*v1.NetworkBaselineStatusPeer, error) {
+	deployment, found, err := m.deploymentDS.GetDeployment(ctx, deploymentID)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Since you have tests with mocks, you could get coverage for these lines, and the other lines that don't have code coverage. However, I think postgesql tests are a higher priority.

Reply:
Have added postgres tests